.github/workflows/cicd-trivy-dependency-scan.yml

```diff
@@ -17,7 +17,7 @@ jobs:
           ignore-unfixed: true
           format: 'sarif'
           output: 'trivy-results.sarif'
-          severity: 'CRITICAL'
+          severity: 'HIGH'
```

Comment:
I believe this value is a definitive list, so needs to be `CRITICAL,HIGH` to include both 🙈